### PR TITLE
Update ice-v1.md

### PR DESCRIPTION
**Pull request by amosr**

### CODE REVIEW

File: doc/design/ice-v1.md [hunk 1]
```diff
@@ -12,31 +12,39 @@ We can ignore a whole bunch of things:
...
-    - Converting maps to/from C values
-2. Array of strings
-    - Converting to/from C values
-3. Melt bufs (tran)
+1. PSV output for:
+    - Maps, arrays, pairs?
+2. PSV input for arrays of primitive types
+    - Strings, Ints?
+    - Can we have arrays inside structs, or structs inside arrays?
+3. PSV missing output for last item
```
Comment (tranma) [inline doc/design/ice-v1.md]: what's this?

Comment (jystic) [inline doc/design/ice-v1.md]: The last entity is never processed because it doesn't trigger the entity rollover

Comment (jystic) [inline doc/design/ice-v1.md]: This is fixed in https://github.com/ambiata/icicle/pull/231

**Comment by amosr** [comment]
This doesn't particularly matter. Merging anyway

**Comment by jystic** [comment]
:+1:

**Pull request merged**